### PR TITLE
sessionctx: support fast analyze session control variable.

### PR DESCRIPTION
**Pull request by lzmhhh123**
### What problem does this PR solve? 
A sub PR for fast analyze. Reference: #9973 

### What is changed and how it works?
Just add a `sessionctx` variable.

### Check List 

Tests 

 - Unit test
 - Integration test

Code changes

 - Has exported variable/fields change
 - Has persistent data change

Side effects

 - Increased code complexity

Related changes

 - Need to update the documentation
 - Need to be included in the release note

### CODE REVIEW

File: sessionctx/variable/varsutil_test.go [hunk 1]
```diff
@@ -79,6 +79,7 @@ func (s *testVarsutilSuite) TestNewSessionVars(c *C) {
 	c.Assert(vars.MemQuotaNestedLoopApply, Equals, int64(DefTiDBMemQuotaNestedLoopApply))
 	c.Assert(vars.EnableRadixJoin, Equals, DefTiDBUseRadixJoin)
 	c.Assert(vars.AllowWriteRowID, Equals, DefOptWriteRowID)
+	c.Assert(vars.EnableFastAnalyze, Equals, DefTiDBUseFastAnalyze)
```
Comment (eurekaka) [inline sessionctx/variable/varsutil_test.go]: Add check in `ValidateSetSystemVar` as well?

Comment (lzmhhh123) [reply sessionctx/variable/varsutil_test.go]: Already added.

File: sessionctx/variable/sysvar.go [hunk 2]
```diff
@@ -686,6 +686,7 @@ var defaultSysVars = []*SysVar{
 	{ScopeSession, TiDBEnableRadixJoin, BoolToIntStr(DefTiDBUseRadixJoin)},
 	{ScopeSession, TiDBCheckMb4ValueInUTF8, BoolToIntStr(config.GetGlobalConfig().CheckMb4ValueInUTF8)},
 	{ScopeSession, TiDBSlowQueryFile, ""},
+	{ScopeSession, TiDBEnableFastAnalyze, BoolToIntStr(DefTiDBUseFastAnalyze)},
```
Comment (lamxTyler) [inline sessionctx/variable/sysvar.go]: What about also make it a global scope variable?

Comment (lzmhhh123) [reply sessionctx/variable/sysvar.go]: Changed.

File: sessionctx/variable/session.go [hunk 3]
```diff
@@ -387,6 +390,7 @@ func NewSessionVars() *SessionVars {
 		L2CacheSize:               cpuid.CPU.Cache.L2,
 		CommandValue:              uint32(mysql.ComSleep),
 		SlowQueryFile:             config.GetGlobalConfig().Log.SlowQueryFile,
+		EnableFastAnalyze:         false,
```
Comment (lamxTyler) [inline sessionctx/variable/session.go]: No need for it? The default value for `bool` is false.

Comment (lzmhhh123) [reply sessionctx/variable/session.go]: Removed. And btw, `builtinGlobalVariable` has also been added.

**Comment by zz-jason** [comment]
@lzmhhh123 we also needs to update the document

**Comment by lzmhhh123** [comment]
@zz-jason Yes, I will update it in the doc.

**Pull request merged**